### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Tables_Table Grid_Keyboard: We are not able to adjust the width of column header via keyboard.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/AzureStorageExplorer#7587]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1

### Regression From

No

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11

### GitHub Tags:
#A11yMAS;#A11yTCS;#Win11;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#Keyboard;#WCAG2.1.1;#FTP;

### Environment Details:

Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)

### Steps to Reproduce

1. Launch Storage Explorer.
2. TAB to "Open Connect dialog" pane and Sign in to Azure.
3. TAB to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. TAB to "Tables" of the selected subscription and expand it.
5. TAB to any data item and press ENTER on it.
6. TAB to column headers present in table.
7. Observe that We are not able to adjust the width of column header via keyboard.

### Actual Experience

We are not able to adjust the width of column header via keyboard.

### Expected Experience

We should be able to adjust the width of column header via keyboard.

### User Impact:

keyboard only users will miss the resize functionality present for the columns headers.

![Width](https://github.com/microsoft/AzureStorageExplorer/assets/115216656/4bb7b258-c005-4519-9961-10817f5fd5e6)

**Comment by craxal** [comment]
Best option here is to add column sizes to the Customize Columns dialog.

**Comment by Rama8345** [comment]
Lowering the severity to Sev 3.

**Comment by craxal** [comment]
This has been fixed by adding the ability to adjust column sizes in the Customize Columns dialog. This can be opened in most explorers by opening the context menu while focused on a column header. The table explorer also has a toolbar item that will open the dialog.

**Comment by kupatkar99** [comment]
@craxal we are unable to adjust the width using the customize column dialog. Could you please help us out with the steps need to follow for the same.

**Comment by craxal** [comment]
@kupatkar99

1. Open a table.
2. Right-click any header in the table and select "Customize Columns..."
3. Or, click the Customize Columns button in the toolbar.
4. Modify column sizes using the number inputs.

![image](https://github.com/microsoft/AzureStorageExplorer/assets/3473026/259414fa-7988-4348-b5aa-3ab682c12ef4)

<img width="505" alt="image" src="https://github.com/microsoft/AzureStorageExplorer/assets/3473026/27992f40-25c4-4aaf-b564-e848d194337e">

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
Version: 1.33.0-dev(96)
Issue is fixed hence Closing the bug Please find added attachment for reference. 
![7587_Fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/3e6740b0-ff27-428c-92a5-7017774512d7)
